### PR TITLE
Fix indexing / Kibana configuration

### PR DESCRIPTION
**Pull request by andreibancioiu**
[PR #2434](https://github.com/ElrondNetwork/elrond-go/pull/2434)

### CODE REVIEW

File: core/indexer/factory/indexerFactory.go [hunk 1]
```diff
@@ -14,6 +15,11 @@ import (
 	"github.com/elastic/go-elasticsearch/v7"
 )
 
+const (
+	withKibanaFolder = "withKibana"
```
Comment (SebastianMarian) [inline core/indexer/factory/indexerFactory.go:19]: withKibanaDir or withKibanaDirectory,  "Folder" is something related to windows only :)

File: core/indexer/factory/indexerFactory.go [hunk 2]
```diff
@@ -14,6 +15,11 @@ import (
...
+	noKibanaFolder   = "noKibana"
```
Comment (SebastianMarian) [inline core/indexer/factory/indexerFactory.go:20]: The same here

File: core/indexer/testdata/noKibana/opendistro.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "index_patterns": [".opendistro-*"],
```
Comment (SebastianMarian) [inline core/indexer/testdata/noKibana/opendistro.json:2]: .opendistro or simple opendistro ?

File: core/indexer/testdata/withKibana/opendistro.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "index_patterns": [".opendistro-*"],
```
Comment (SebastianMarian) [inline core/indexer/testdata/withKibana/opendistro.json:2]: .opendistro or simple opendistro ?

**Comment by andreibancioiu** [comment]
Not applicable (merged some time ago into development).

**Pull request closed without merging**